.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   lts:
```

Comment:
@lholmquist looks good, however a nit... could you remove the `lts` label from this matrix job? It ends up appearing as though all test runs are somehow LTS.

![image](https://user-images.githubusercontent.com/15952/219132853-db39d487-a51f-4a91-9185-3191ea3c4386.png)

Reply:
i changed it to build, which is consistent with the other nodeshift org repos